pyproject.toml

```diff
@@ -35,63 +35,39 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-dev = [
-  "black",
-  "build",
-  "flake8",
-  "flakeheaven",
-  "flake8-bandit",
-  "flake8-bugbear",
-  "flake8-builtins",
-  "flake8-comprehensions",
-  "flake8-docstrings",
-  "flake8-simplify",
-  "flake8-use-fstring",
-  "pep8-naming",
-  "pre-commit",
-]
 doc = [
-  "m2r2",
-  "sphinxcontrib-apidoc",
-  "sphinx_rtd_theme",
+  "m2r2==0.3.3.post2",
+  "sphinxcontrib-apidoc==0.3.0",
+  "sphinx_rtd_theme==1.2.0",
 ]
-test = [
-  "pytest-cov",
-  "pytest-subprocess",
+dev = [
+  "pytest-cov==4.0.0",
+  "pytest-subprocess==1.5.0",
 ]
 
 [project.scripts]
 incipyt = "incipyt.__main__:main"
 
-[tool.flakeheaven]
-exclude = [
-  ".env",
-  ".git",
-]
-
-[tool.flakeheaven.plugins]
-flake8-bandit = ["+*", "-S101", "-S404", "-S603", "-S607"]
-flake8-bugbear = ["+*"]
-flake8-builtins = ["+*"]
-flake8-comprehensions = ["+*"]
-flake8-docstrings = ["+*", "-D105", "-D107"]
-flake8-simplify = ["+*"]
-flake8-use-fstring = ["+*", "-FS003"]
-pep8-naming = ["+*"]
-pycodestyle = ["+*", "-W503", "-E501"]
-pyflakes = ["+*"]
+[tool.black]
+line-length = 120
```

Comment:
It is always very hard to justify "what line-lenght shall we use" and there are always a lot of discussions about it. I suggest we use the following metric: the code should look alike in the IDE and on Github on a 1920x1080 screen with the default zoom (100%). When reviewing a PR in split-mode on github (when the editor is the narrowest), the editor is ~100 characters wide. I suggest we use that value as line-length.

I've a good example here, the first line is 98 characters long and fit in one line, the second is 99 characters long and needs two lines on Github.

![image](https://user-images.githubusercontent.com/8208953/229351123-ecb107d7-5924-48e5-877c-04fb375f051d.png)

So here I'm suggesting 100 but maybe it should be 99 or 98 instead, you can use those two lines as test value :)

```suggestion
line-length = 100
```